0a.md

```diff
@@ -0,0 +1,66 @@
+NIP-XX
+======
+
+Audio Events
+---------------
+
+`draft` `optional`
+
+This NIP defines several new event kinds representing different types of audio track. These events
+are _parameterized replaceable_ as defined in [NIP-01](./01.md) and deletable per [NIP-09](09.md).
+
+Unlike a `kind 1` event with audio attached, audio events are intended to be surfaced in an audio-specific context
+rather than in a general micro-blogging context.
+
+The following kinds are defined by this NIP:
+
+- `31337` represents a music track
+- `31338` represents a podcast episode
+
+The following tags are required:
+
+- `d` is a unique identifier randomly generated by the client.
+- `media` is the url of the media. This SHOULD NOT be read, but is required for backwards compatibility reasons.
+- `imeta` is as described in [NIP 92](./92.md). Multiple `imeta` tags MAY be included to provide for different
+  use cases (e.g. streaming vs download).
+- `title` is the title of the audio track
+- `subject` is the title of the audio track (deprecated, but required for compatibility)
+
+The following tags are optional:
+
+- `i` is an external GUID in the format defined [here](https://github.com/MerryOscar/nips/commit/280eb498e0ac56b8f9356c1b7a88cc8b31579801).
+- `c` is the track's value as defined by a given category.
+- `website` is an external url to a website related to the track.
+- `duration` is the duration of the audio track, in seconds.
+- `published_at` is a timestamp representing the track's original publish date.
+
```

Comment:
```suggestion
- `price` is the value + UNIT, ie `[ "price", "1000", "SATS" ]
```

Reply:
This seems like it belongs somewhere else, since it may not be unique to audio events, and requires a lot of context to be used properly. Maybe draft up a new NIP that describes what fanfares is doing?

Reply:
Sure, I just copied NIP-99 but we don't need to include it here.

Reply:
Yeah, I think there's more complexity involved with selling content vs other stuff, since I assume the content would be paywalled, and there would need to be a mechanism for unlocking it.